www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -806,19 +800,32 @@ function (declare, domClass, dom, style, fx, Toggler, keys, on, query, Memory,
     _formatReportDetails : function (report, reportDetails) {
       var res = [];
 
+      var lastBugEvent = null;
+      if (reportDetails.pathEvents.length) {
+        lastBugEvent = reportDetails.pathEvents[reportDetails.pathEvents.length-1]
+      }
+
       res.push({
         id : report.reportId + '_0',
         name : '<b><u>' + entities.encode(report.checkerMsg) + '</u></b>',
         parent : report.reportId + '',
         report : report,
         isLeaf : true,
         idx : reportDetails.pathEvents.length,
-        kind : 'msg'
+        kind : 'msg',
+        bugPathEvent : lastBugEvent
       });
 
-      if (reportDetails.pathEvents.length <= 1)
+      if (!lastBugEvent)
         return res;
 
+      setTimeout(function () {
+          that.editor.highlightBugPathEvent({
```

Comment:
I think `that` is not defined here.
https://github.com/Ericsson/codechecker/blob/30436f49de0a6cfacf6d3204eb516a2ce136c275/www/scripts/codecheckerviewer/BugViewer.js#L845